test/integration/controller/jobs/jobset/jobset_controller_test.go

```diff
@@ -159,10 +159,10 @@ var _ = ginkgo.Describe("JobSet controller", ginkgo.Ordered, ginkgo.ContinueOnFa
 		}, util.Timeout, util.Interval).Should(testing.BeNotFoundError())
 		// check the original wl is still there
 		//gomega.Expect(k8sClient.Get(ctx, wlLookupKey, createdWorkload)).Should(gomega.Succeed())
-		gomega.Consistently(func() bool {
+		gomega.Eventually(func() bool {
 			err := k8sClient.Get(ctx, wlLookupKey, createdWorkload)
 			return err == nil
-		}, util.ConsistentDuration, util.Interval).Should(gomega.BeTrue())
+		}, util.Timeout, util.Interval).Should(gomega.BeTrue())
```

Comment:
```suggestion
		gomega.Eventually(func() error {
			return k8sClient.Get(ctx, wlLookupKey, createdWorkload)
		}, util.Timeout, util.Interval).Should(gomega.Succeeded())
```

Please check the other places.